notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -31,19 +31,23 @@
+    "Hopefully, this will not only make it much easier to generate these files, but also keep our information more accurate and tractable.\n",
     "\n",
     "### Here's how it should work.\n",
     "\n",
-    "You'll start out with a **basic plate map**, which just links each sample to it's approprite row and column.\n",
+    "You'll start out with a **sample accession file**, which links each sample to it's approprite matrix tube barcode. Then you'll read in the sample information from a Qiita metadata file. \n",
     "\n",
-    "Then you'll add the output of the MiniPico assay of sample DNA concentrations, which will enable to you to automatically make a **normalization pick list** for starting the shotgun library prep itself. You can also visualize these concentrations on the plate, allowing you to double check the plate map and DNA concentration read.\n",
+    "Next, you'll use a Compression Layout form to generate a 384-well dataframe of the sample well-locations and assocaited extraction plate metadata [`Project Name`, `Project Plate`, `Project Abbreviation`, `Plate elution volume`] The Compression Layout form will import 96-well plate layouts from **VisionMate plate reader output files**, and will assign a 384-well sample well-location based on the 384-well quadrant (`Plate Position`) each 96-well plate is occupying. Then, you'll compare the matrix tube barcodes in your 384-well plate with those stored in folders documenting control matrix tubes to automatically assign controls using the add_controls() function, followed by a validation step. \n",
     "\n",
-    "Next you'll automatically assign barcodes to each sample, producing an **index pick list** for barcode addition prior to PCR.\n",
+    "Next, you will merge the **quantification file** of the MiniPico assay of sample DNA concentrations, which will enable to you to automatically make a **normalization pick list** for starting the shotgun library prep itself. You can also visualize these concentrations on the plate, allowing you to double check the plate map and DNA concentration read.\n",
     "\n",
-    "After finishing the shotgun library prep itself, you'll measure library concentration with qPCR. This qPCR data file will then be read in and used to estimate and visualize pooling parameters, producing a **pooling pick list**. \n",
+    "Next you'll automatically assign barcodes to each sample by specifying a **plate counter**, producing a unique **index pick list** for barcode addition prior to PCR.\n",
     "\n",
-    "Finally, the per-sample information from the whole run can be combined to automatically produce a **sample sheet** that you can give directly to IGM for sequencing. "
+    "After finishing the shotgun library prep itself, you'll measure library concentration with the MiniPico assay. The **quantification file** will then be read in and used to estimate and visualize pooling parameters, producing a **pooling pick list**. \n",
```

Comment:
Just checking: this is a *different* quantification file than the one referenced above, right?  I see **quantification file** twice close together, and my immediate thought is that both those refer to the same thing, but I think they don't ...

Reply:
They are different quantifications. The first quantification file quantifies gDNA concentration, while the second one quantifies sequencing library concentration. I can make a clarification.